lib/token.h

```diff
@@ -453,6 +453,12 @@ class CPPCHECKLIB Token {
     unsigned char bits() const {
         return mBits;
     }
+    bool hasPointer() const {
```

Comment:
If this is only a special flag for `TemplateSimplifier` I think we can have a more complex name.. how about `isTemplateSimplifierInstantiationPointer`

Reply:
Fixed.  Renamed to hasTemplateSimplifierPointer because declaration pointers will be supported in later patches and token is not a pointer but has a pointer to it.